File: packages/insomnia-httpsnippet/test/fixtures/output/ruby/native/text-plain.rb
require 'uri'
require 'net/http'

url = URI("http://mockbin.com/har")

http = Net::HTTP.new(url.host, url.port)

request = Net::HTTP::Post.new(url)
request["content-type"] = 'text/plain'
request.body = "Hello World"

response = http.request(request)
puts response.read_body

File: packages/insomnia-httpsnippet/test/fixtures/output/node/native/text-plain.js
var http = require('http');

var options = {
  method: 'POST',
  hostname: 'mockbin.com',
  port: null,
  path: '/har',
  headers: {
    'content-type': 'text/plain',
  },
};

var req = http.request(options, function(res) {
  var chunks = [];

  res.on('data', function(chunk) {
    chunks.push(chunk);
  });

  res.on('end', function() {
    var body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.write('Hello World');
req.end();
